examples/svm/plot_separating_hyperplane.py

```diff
@@ -12,37 +12,39 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from sklearn import svm
-
+from sklearn.datasets import make_classification
 # we create 40 separable points
 np.random.seed(0)
-X = np.r_[np.random.randn(20, 2) - [2, 2], np.random.randn(20, 2) + [2, 2]]
-Y = [0] * 20 + [1] * 20
 
+X, Y = make_classification(n_features=2, n_redundant=0, n_informative=1,
```

Comment:
You should use `y` here. The scikit-learn convention is that variable names starting with a capital letter should be reserved for 2d arrays.